pinax/announcements/urls.py

```diff
@@ -0,0 +1,13 @@
+from django.conf.urls import patterns, url
+
+from pinax.announcements import views
+
+urlpatterns = patterns(
```

Comment:
`patterns(` is no longer supported.  Should just be a list of `url()` declarations.